cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -119,6 +119,22 @@
   php_versions.map { |v| [v.split('.')[0, 2].join('.'), v] }
 ]
 
+nodejs_versions = %w(
+  7.4.0
```

Comment:
The versions listed here will mostly be for show/example, as the ones actually installed are specified at the packer-templates level.